source/user-manual/manager/configuring-cluster/advanced-settings.rst

```diff
@@ -1,27 +1,735 @@
+                    ;;
+                esac
+                exit $ret
+                ;;
+            *)
+                echo "Usage: /etc/init.d/${BASENAME} {start|stop|reload|restart|status}"
+                exit 2
+                ;;
+            esac
+
+            :
+
+
+        .. raw:: html
+
+            </details>
+
+        And a :ref:`Configuration file <haproxy_configuration>` to get the service up and running.
+
+        1. It will be needed to put these files in the same directory and build the image
+
+        .. code-block:: console
+
+            # tree
+            .
+            ├── Dockerfile
+            ├── entrypoint.sh
+            ├── haproxy.cfg
+            └── haproxy-service
+
+        .. code-block:: console
+
+            # docker build --tag=haproxy-deploy .
+
+        2. After building the image can we run the haproxy service
+
+        .. code-block:: console
+
+            # docker run haproxy-deploy
+            TCPLOG: true HTTPLOG: true
+            * Starting haproxy haproxy                                                                                                                                                                                          [NOTICE]   (33) : haproxy version is 2.8.9-1842fd0
+            [NOTICE]   (33) : path to executable is /usr/sbin/haproxy
+            [ALERT]    (33) : config : parsing [/etc/haproxy/haproxy.cfg:3] : 'pidfile' already specified. Continuing.
+
+
+.. _haproxy_configuration:
+
+Configuration
+-------------
+
+    1. The configuration must be put into ``/etc/haproxy/haproxy.cfg``.
+
+        .. raw:: html
+
+            <details>
+            <summary><b>haproxy.cfg</b></summary>
+
+        .. code-block:: cfg
+
+            global
+                chroot      /var/lib/haproxy
+                pidfile     /var/run/haproxy.pid
+                maxconn     4000
+                user        haproxy
+                group       haproxy
+                stats socket /var/lib/haproxy/stats level admin
+                log 127.0.0.1 local2 info
+
+            defaults
+                mode http
+                maxconn 4000
+                log global
+                option redispatch
+                option dontlognull
+                option tcplog
+                timeout check 10s
+                timeout connect 10s
+                timeout client 1m
+                timeout queue 1m
+                timeout server 1m
+                retries 3
+
+            frontend wazuh_register
+                mode tcp
+                bind :1515
+                default_backend wazuh_register
+
+            backend wazuh_register
+                mode tcp
+                balance leastconn
+                server master <IP_OR_DNS_OF_WAZUH_MASTER_NODE>:1515 check
+                server worker1 <IP_OR_DNS_OF_WAZUH_WORKER_NODE>:1515 check
+                server workern <IP_OR_DNS_OF_WAZUH_WORKER_NODE>:1515 check
+
```

Comment:
The current configuration is fine and will allow agents to register, but once registered (on port 1515) they won't be able to connect to any manager since it happens on port 1514. An additional frontend and backend is required for this, but only if the HAProxy helper is not used. If the helper is used, it will add the `wazuh_reporting` frontend and backend (the name will depend on the `<haproxy_backend>` option):

```suggestion
            frontend wazuh_reporting_front
                mode tcp
                bind :1514 name wazuh_cluster_front_bind
                default_backend wazuh_cluster

            backend wazuh_reporting
                mode tcp
                balance leastconn
                server master <IP_OR_DNS_OF_WAZUH_MASTER_NODE>:1514 check
                server worker1 <IP_OR_DNS_OF_WAZUH_WORKER_NODE>:1514 check
                server worker2 <IP_OR_DNS_OF_WAZUH_WORKER_NODE>:1514 check

```

It might be useful to highlight the lines and explain to the user that this part should not be included if they intend to use the helper. 

Or maybe the other way around. Do not add it to `haproxy.cfg` and explain/share the block that should be included if the helper is not used.